BedrockCommand.cpp

```diff
@@ -146,6 +143,16 @@ void BedrockCommand::stopTiming(TIMING_INFO type) {
     get<2>(_inProgressTiming) = 0;
 }
 
+bool BedrockCommand::AreHttpsRequestsComplete() const {
```

Comment:
Eek, sorry for the nitpick, but lowerCamelCase please 😬

Reply:
Yes, I don't know why I did that.